shellbase-core/src/main/scala/com/sumologic/shellbase/ShellCommand.scala

```diff
@@ -132,6 +147,11 @@ abstract class ShellCommand(val name: String,
     None
   }
 
+  def postInformationToSlackThread(ts: String, commandExecuteTimeDuration: Long, commandResult: Boolean):
+  Option[String] = {
```

Comment:
This looks awkward on this line - should we put it back on the previous line?  If the goal is better spacing, we should do line breaks between arguments instead